torch/_dynamo/variables/dicts.py

```diff
@@ -128,8 +128,17 @@ def __eq__(self, other: "ConstDictVariable._HashableTracker") -> bool:
             return Hashable._eq_impl(self.underlying_value, other)
 
     def __init__(
-        self, items: Dict[VariableTracker, VariableTracker], user_cls=dict, **kwargs
+        self,
+        items: Dict[VariableTracker, VariableTracker],
+        user_cls=dict,
+        is_python_arg=False,
+        **kwargs,
     ) -> None:
+        if "original_items" in kwargs:
+            kwargs.pop("original_items")
+        if "pop_items" in kwargs:
+            kwargs.pop("pop_items")
```

Comment:
Where are these coming from?

Reply:
From a few lines below. These are needed because `.clone()` will create a copy of `ConstDict` with these variables as arguments and `super().__init__(**kwargs)` will fail